static/app/components/events/interfaces/crashContent/exception/utils.spec.tsx

```diff
@@ -122,4 +122,19 @@ describe('Linkify()', function () {
 
     expect(container.firstChild).toHaveTextContent('Go to');
   });
+
+  it('applies links to text containing a single url and text and ignores a period', function () {
+    // fix
+    const url = 'https://www.example.com';
```

Comment:
There's a case this doesn't cover, which is URLs like `https://example.com/page.html`

We probably only want to ignore the period when there is nothing but whitespace after it